libass/ass_utils.h

```diff
@@ -179,9 +179,9 @@ static inline uint32_t fnv_32a_buf(void *buf, size_t len, uint32_t hval)
     size_t n = (len + 3) / 4;
 
     switch (len % 4) {
-    case 0: do { hval ^= *bp++; hval *= FNV1_32A_PRIME;
-    case 3:      hval ^= *bp++; hval *= FNV1_32A_PRIME;
-    case 2:      hval ^= *bp++; hval *= FNV1_32A_PRIME;
+    case 0: do { hval ^= *bp++; hval *= FNV1_32A_PRIME; //-fallthrough
```

Comment:
Is there any special meaning in `-` symbol? As far as I know, simple `// falthrough` is sufficient for gcc/clang.

Reply:
See https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wimplicit-fallthrough (as linked in the OP). There’s a warning level option and this works at the highest level whereas simple `fallthrough` doesn’t.